sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/SqlScriptingParserSuite.scala

```diff
@@ -57,7 +61,7 @@ class SqlScriptingParserSuite extends SparkFunSuite with SQLHelper {
 
   test("multi select") {
     val sqlScriptText = "BEGIN SELECT 1;SELECT 2; END"
```

Comment:
how about `SELECT 1;SELECT 2;` without BEGIN END? will it fail?

Reply:
Added.